src/components/token/TokenAlerts.js

```diff
@@ -10,7 +10,7 @@ const TokenAlerts = (props) => {
   }, [props.token]);
 
   const bannedAlert = () => {
-    if (!token.banned) {
+    if (!token.meta || !token.meta.data.banned) {
```

Comment:
This expects the token to have `meta.data`, right?

What if the token has the old structure?

Reply:
It will just evaluate as false, as the old structure doesn't have `meta` property